packages/workbox-background-sync/src/lib/QueueStore.ts

```diff
@@ -183,38 +159,14 @@ export class QueueStore {
    * @private
    */
   async _removeEntry({direction}: {direction?: IDBCursorDirection}) {
-    const [entry] = await this._db.getAllMatching(OBJECT_STORE_NAME, {
-      direction,
-      index: INDEXED_PROP,
-      query: IDBKeyRange.only(this._queueName),
-      count: 1,
-    });
+    const entry = await this._queueDb.getEndEntryFromIndex(
```

Comment:
See my earlier comment about the method signature for `getEndEntryFromIndex()`, as passing in the first parameter wrapped in `{}` looks awkward.